src/components/Stack/tests/Stack.test.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Stack from '../Stack';
+import Badge from '../';
+import Item from '../Item';
+import Heading from '../';
+
+describe('Stack Component Suit', () => {
+    describe('Test children property', () => {
+        it('should verify children property is defined', () => {
+            const stackWrapper = mount(
+                                        <Stack>
+                                          <Badge>Paid</Badge>
+                                          <Badge>Fulfilled</Badge>
+                                        </Stack>,
```

Comment:
not sure why the comma is there. mount doesn't take a second argument (this applies to every single one)